cloudflare/resource_cloudflare_page_rule_test.go

```diff
@@ -958,6 +988,17 @@ resource "cloudflare_page_rule" "test" {
 }`, zoneID, target, zoneName)
 }
 
+func testAccCheckCloudflarePageRuleConfigDisableZaraz(zoneID, target string) string {
+	return fmt.Sprintf(`
+resource "cloudflare_page_rule" "test" {
+	zone_id = "%s"
+	target = "%s"
+	actions {
+		disable_zaraz = true
+	}
+}`, zoneID, target)
+}
+
```

Comment:
I ran it and checked on the dashboard and it worked as expected :
![image](https://user-images.githubusercontent.com/29628106/159140510-af4dd236-e2c8-48ea-95b5-bdb068a560f2.png)